addons/customGear/fauxWeapons/config.cpp

```diff
@@ -81,4 +106,21 @@ class CfgWeapons {
             maxRangeProbab = 0.05;
         };
     };
-};
\ No newline at end of file
+    
+    // Adds MG5 in 7.62x51mm
+    class MMG_01_tan_F;
+    class GVAR(MG5): MMG_01_tan_F {
```

Comment:
You shouldn't generally change a class name that some one might be using once it actually enters the public version of a mod, this includes the C7 (currently `potato_fauxC7_c7`) and MG5 (currently `potato_fauxMG5_MG5`). That's also why I was asking if you'd be willing to choose a different prefix here https://github.com/BourbonWarfare/POTATO/pull/590#pullrequestreview-2393983513.

Reply:
i think i fixed this now, but i agree - i'll update the kac lamg branch to have a set classname following the "potato_(weaponclass)_(weaponname)" format